buildbot/master/files/config/factories.py

```diff
@@ -115,19 +115,17 @@ def make_step(self, command):
 ])
 
 
-def make_cd_string(command):
-    return "cd /c/buildbot/slave/windows/build ; " + command
+def make_cd_command(command):
```

Comment:
This should be renamed to express the intent of the function, instead of what it is doing. Let's use `make_win_command`, since this function takes a regular command and makes it palatable to Windows.